manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: FirebirdProject.Firebird.4
+PackageVersion: 4.0.3.2975
+PackageLocale: en-US
+Publisher: Firebird Project
+PublisherUrl: https://firebirdsql.org/en/start/
+PublisherSupportUrl: https://firebirdsql.org/en/support/
+Author: Firebird Project
+PackageName: Firebird
+PackageUrl: https://firebirdsql.org/en/firebird-4-0/
+License: Initial Developer's Public License Version 1.0
+LicenseUrl: https://firebirdsql.org/en/initial-developer-s-public-license-version-1-0/

```

```suggestion
LicenseUrl: https://github.com/FirebirdSQL/firebird/#licensing
```

This URL provides the user with more context about licensing